contracts/src/v0.8/dev/interfaces/OCR2DRRegistryInterface.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+/**
+ * @title OCR2DR billing interface.
+ */
+interface OCR2DRRegistryInterface {
+  struct RequestCommitment {
+    uint64 blockNum;
+    uint64 subId;
+    uint32 callbackGasLimit;
+    bytes data;
+    address sender;
+  }
+
+  /**
+   * @notice Get configuration relevant for making requests
+   * @return minimumRequestConfirmations global min for request confirmations
+   * @return maxGasLimit global max for request gas limit
+   * @return s_provingKeyHashes list of registered key hashes
```

Comment:
Having this identifier `s_provingKeyHashes` looks odd.. it does not appear anywhere here and hence confusing. Maybe we should change @ return documentation to not use any identifiers here?